app/views/users/profile.html.erb

```diff
@@ -247,7 +247,7 @@
 
   <div class="info float:left">
     <ul>
-      <li class="mt-3"><h5>@<%= @profile_user.name %> <%= @profile_user.new_contributor %> asked</h5></li>
+    <li class="mt-3"><h5>@<%= @profile_user.name %> has</h5></li> 
```

Comment:
I see you did a dedent of 2 whitespaces for the `li`, let's shift it back. Tha typo changes looks good. PR is good to go once you shift the `li` back to its position by indenting it by 2 whitespaces.

Reply:
ok thnx @Anupam-dagar

Reply:
Ok I have again pushed

Reply:
I can't see the new changes. Please check.